server/src/main/java/io/seata/server/Server.java

```diff
@@ -80,7 +80,8 @@ public static void main(String[] args) throws IOException {
         // register ShutdownHook
         ShutdownHook.getInstance().addDisposable(coordinator);
         ShutdownHook.getInstance().addDisposable(rpcServer);
-
+        //IdWorker Instantiate by service node
+        IdWorker.getInstance(parameterParser.getServerNode());
```

Comment:
How about change to Idworker.init(parameterParser.getServerNode())

Reply:
PTAL @l81893521